software/metax/genotype/CYVCF2Genotype.py

```diff
@@ -8,7 +8,7 @@
 
 def vcf_file_geno_lines(path, mode="genotyped", variant_mapping=None, whitelist=None, skip_palindromic=False, liftover_conversion=None):
     logging.log(9, "Processing vcf %s", path)
-    vcf_reader = VCF(path)
+    vcf_reader = VCF(path, gts012=True)
```

Comment:
What does gts012 represent?

Reply:
From cyvcf2.VCF documentation:
gts012 (bool) – if True, then gt_types will be 0=HOM_REF, 1=HET, 2=HOM_ALT, 3=UNKNOWN. If False, 3, 2 are flipped.